src/scenes/CreateEvent/store.js

```diff
@@ -171,6 +177,32 @@ export default class CreateEventStore {
     );
   }
 
+  @action
+  onEnterDialog = async () => {
+    try {
+      const { data: { result } } = await axios.post(
```

Comment:
Why is this logic duplicated in two places? it is the same logic as `prepareToCreateEvent()`. We don't need to call this logic twice.